go-controller/pkg/clustermanager/clustermanager.go

```diff
@@ -102,6 +104,11 @@ func NewClusterManager(ovnClient *util.OVNClusterManagerClientset, wf *factory.W
 			return nil, err
 		}
 	}
+	if config.Kubernetes.OVNEmptyLbEvents {
+		if _, err := unidling.NewUnidledAtController(&kube.Kube{KClient: ovnClient.KubeClient}, wf.ServiceInformer()); err != nil {
```

Comment:
is CM's watch factory initializing/creating the services informer? I can't remember what we use that for?

Reply:
It is: https://github.com/ovn-org/ovn-kubernetes/blob/6deaf7a26d5dfb0ce56cc4b0dc741508c6bbd679/go-controller/pkg/factory/factory.go#L505
EgressService controller is using it: https://github.com/ovn-org/ovn-kubernetes/blob/96b78039b210bfbadcf39de1ed468f9cd0328c99/go-controller/pkg/clustermanager/egressservice/egressservice_cluster.go#L131